docs/user_guide/using_the_library.rst

```diff
@@ -55,6 +55,41 @@ This has the same return structure as :py:class:`~nwbinspector.nwbinspector.insp
 
 
 
+.. _simple_streaming_api:
+
+Inspect a DANDI set (ROS3)
+--------------------------
+
+It is a common use case to want to inspect and review entire datasets of NWB files that have already been
+uploaded to the :dandi-archive:`DANDI Archive <>`. While it is possible to simply download the entire dandiset to your local computer and
+run the NWB Inspector as usual, there is an alternative which is less expensive in terms of bandwith. This can be especially
+useful when the DANDI set is large and impractical to download in full, such as one that is multiple terabytes in total size.
+
+A general tutorial for using the :code:`ros3` driver can be found in the :ros3-tutorial:`PyNWB documentation <>` - however, the NWB Inspector has automated functionality for resolving the technical identification of assets such that the only piece of information needed is the six-digit DANDI set identifier...
```

Comment:
careful with the word "however". I don't think it's appropriate in this case.

Reply:
Fixed